src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.es.xlf

```diff
@@ -74,12 +74,12 @@
       </trans-unit>
       <trans-unit id="LocalizedControlTypeHyperlink">
         <source>hyperlink</source>
-        <target state="translated">hipervínculo</target>
+        <target state="translated">hyperlink</target>
```

Comment:
LocalizedControlType* strings are used to register control types for UI Automation.  These strings should be localized.